config/.env.default

```diff
@@ -17,8 +17,8 @@ export APP_DEFAULT_LOCALE="en_US"
 export SECURITY_SALT="__SALT__"
 
 # Uncomment these to define BEDITA API base URL and API KEY
-export BEDITA_API="https://bedita-api-url"
-export BEDITA_API_KEY="bedita-api-key"
+# export BEDITA_API="https://bedita-api-url"
```

Comment:
This `.env.default` file will be copied in `.env` and those env vars are mandatory, as explained in README.md => I would leave them uncommented (they are not optional).